src/DGSM.jl

```diff
@@ -53,7 +54,14 @@ function DGSM(f,samples::Int,distr::AbstractArray, crossed::Bool = false)
     asq = [mean(dfdx[:,i].^2) for i in 1:k]
     absa = [mean(abs.(dfdx[:,i])) for i in 1:k]
     
-    
+    sigma = zeros(Float64,k)
+
+    for i in 1:k
+        for j in 1:samples
+            sigma[i] += 0.5*(XX[j][i])*(1-XX[j][i])*dfdx[j + (i-1)*samples]^2
```

Comment:
is this supposed to be xi equation 5?

Reply:
Yes

Reply:
Why was the name changed? This quantity should definitely be described in docs

Reply:
Which docs are you referring to?